src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -129,5 +129,17 @@
             <ToggleSwitch IsOn="{x:Bind ViewModel.AutoHideWindow, Mode=TwoWay}"
                           Style="{StaticResource ToggleSwitchInExpanderStyle}" />
         </local:SettingContainer>
+
+        <!--  Show Admin Shield  -->
+        <local:SettingContainer x:Uid="Globals_ShowAdminShield">
+            <ToggleSwitch IsOn="{x:Bind ViewModel.ShowAdminShield, Mode=TwoWay}"
+                          Style="{StaticResource ToggleSwitchInExpanderStyle}" />
+        </local:SettingContainer>
+
+        <!--  Enable Unfocused Acrylic  -->
+        <local:SettingContainer x:Uid="Globals_EnableUnfocusedAcrylic">
+            <ToggleSwitch IsOn="{x:Bind ViewModel.EnableUnfocusedAcrylic, Mode=TwoWay}"
+                          Style="{StaticResource ToggleSwitchInExpanderStyle}" />
+        </local:SettingContainer>

```

Comment:
Noticed that this setting is `compatibility.enableUnfocusedAcrylic`. Any ideas why it's in the compatibility namespace? To me, it makes sense to have in the appearance page, but the `compatibility` namespace is making me second guess that it should be in the new compatibility page. Should we remove the `compatibility` prefix?

Reply:
I think it was more "OS compatibility" - unfocused acrylic is actually a different implementation of acrylic than usual. It has caused some issues in the past.